contrib/guix/manifest.scm

```diff
@@ -158,7 +158,7 @@ chain for " target " development."))
 (define-public python-lief
   (package
     (name "python-lief")
-    (version "0.13.2")
+    (version "0.16.5")
```

Comment:
Does Guix build succeed?

Reply:
In the process of testing this, I'll mark as draft in the meanwhile.